docs/docs/configuration/systask.md

```diff
@@ -548,6 +622,45 @@ The task output can then be referenced in downstream tasks like:
 }
 ```
 
+## Inline Task
+
```

Comment:
same as above: `INLINE` to replace `LAMBDA`

Reply:
Done.